### DISCUSSION TITLE
How to prompt for general exploration

### DISCUSSION THREAD

**Discussion by DSA101** [AI4Finance-Foundation/FinGPT#112] [Q&A]
Is there an example or a guide how to prompt the model for general questions and exploration? E.g. "Who are the main competitors of the Carnival Cruises?", "Who are the main customers of Taiwan Semiconductor?".

**Comment by dwniu2000** [comment]
ask GPT4 or check Langchain for how to  use local RAG database

**Comment by artist-genai** [comment]
Did you ever find an answer to this question?
